examples/package.json

```diff
@@ -73,6 +73,7 @@
     "react-helmet": "5.2.0",
     "react-measure": "2.0.2",
     "react-router-dom": "4.2.2",
+    "react-select": "^1.2.1",
```

Comment:
unnecessary ^

Reply:
VBO confirmed proper use of ^

Reply:
Sure, but the usage of striska is wanted in packages, that clients will add to their own projects, which is not the case of examples for react-components